### PR TITLE
Autofill improvements

### PR DESCRIPTION
**Pull request by soeren-schmaljohann-2denker**
I did't see a ticket for this, but I can create one, if I didn't miss it

### CODE REVIEW

**Review by ABausG — COMMENTED** [review]
Some minor points:
Not sure if it makes sense to do in this PR or create a new ticket/branch as it is going in the same direction:
Improving the Keyboard Actions so that hitting the action button on the softkeyboard moves the focus to the next field (and show next as the type) and a go action if in the last field that performs the call and closes the keyboard.

Possibly add a test for the change in error behaviour. I guess this was done to prevent mail leakage?

Increase in the version is required

**Comment by soeren-schmaljohann-2denker** [comment]
> Some minor points: Not sure if it makes sense to do in this PR or create a new ticket/branch as it is going in the same direction: Improving the Keyboard Actions so that hitting the action button on the softkeyboard moves the focus to the next field (and show next as the type) and a go action if in the last field that performs the call and closes the keyboard.
> 
> Possibly add a test for the change in error behaviour. I guess this was done to prevent mail leakage?
> 
> Increase in the version is required

- I would see that in another ticket, I'll write it

- What error behaviour are you talking about, I don't see any changes to that in this PR.

- I increased the version

**Review by ABausG — COMMENTED** [review]
Forgot to submit the comment reagarding the changed error. Sorry!

File: packages/apptive_grid_user_management/lib/src/login_content.dart [hunk 1]
```diff
@@ -81,84 +81,87 @@ class _LoginContentState extends State<LoginContent> {
...
                       ),
                     ),
-                    if (_error is Response)
+                    if (_error != null) ...[
+                      SizedBox(
+                        height: spacing,
+                      ),
                       Text(
-                        (_error as Response).body,
+                        localization.errorLogin,
```
[the remark this answers is not in the document]
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_user_management/lib/src/login_content.dart:125]: Ah that is not a change, git just jumbled it up. This is form line 123 before the changes @ABausG

[the remark this answers is not in the document]
Comment (ABausG) [reply packages/apptive_grid_user_management/lib/src/login_content.dart:125]: Ah I see. Wow this file got really messed up :D Hiding whitespaces really helped to review this. I guess hiding whitespaces should really be a on option for flutter code all the time to manage wrapping things in new widgets

**Review by ABausG — APPROVED** [review]

**Pull request merged**